rca/landingpages/models.py

```diff
@@ -1,17 +1,242 @@
+    image = models.ForeignKey(
+        "images.CustomImage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    subtitle = models.TextField(max_length=120, blank=True)
+    description = models.TextField(max_length=250, blank=True)
+
+    panels = [
+        FieldPanel("title"),
+        ImageChooserPanel("image"),
+        FieldPanel("subtitle"),
+        FieldPanel("description"),
+    ] + LinkFields.panels
+
+    class Meta:
+        abstract = True
+
+
+class LandingPageFeaturedImage(FeaturedImage):
+    source_page = ParentalKey("LandingPage", related_name="featured_image")
+
+
+class LandingPageRelatedPagegrid(RelatedPage):
+    source_page = ParentalKey(
+        "landingpages.LandingPage", related_name="related_pages_grid"
+    )
+    panels = [PageChooserPanel("page")]
+
+
+class LandingPageRelatedPageHighlights(RelatedPage):
+    source_page = ParentalKey(
+        "landingpages.LandingPage", related_name="related_pages_highlights"
+    )
+    panels = [PageChooserPanel("page")]
 
 
 class LandingPage(BasePage):
-    template = "patterns/pages/landingpage/landing_page.html"
+    """ Defines all the fields we will need for the other versions of landing pages
+    visibility of some extra fields that aren't needed on certain models which inherit LandingPage
+    are controlled at the content_panels level"""
+
+    template = "patterns/pages/landingpage/landing_page--generic.html"
+    hero_image = models.ForeignKey(
+        "images.CustomImage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    hero_colour_option = models.PositiveSmallIntegerField(choices=(HERO_COLOUR_CHOICES))
+    introduction = models.TextField(max_length=500, blank=True)
+    about_page = models.ForeignKey(
+        "wagtailcore.Page",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    highlights_title = models.TextField(max_length=80, blank=True)
+    highlights_link_url = models.URLField(
+        blank=True, help_text=("Optional link to more information")
+    )
+    highlights_link_text = models.CharField(
+        max_length=80, blank=True, help_text=_("The text to display for the link")
+    )
+
+    related_pages_title = models.TextField(
+        max_length=80,
+        blank=True,
+        help_text=_("The title to be displayed above the related pages grid"),
+    )
+    related_pages_text = models.TextField(
+        max_length=250,
+        blank=True,
+        help_text=_(
+            "The brief paragraph of text to be displayed above the related pages grid"
+        ),
+    )
+    page_list_title = models.TextField(
+        max_length=80,
+        blank=True,
+        help_text=_("The title to be displayed above the page list blocks"),
+    )
+    page_list = StreamField([("page_list", RelatedPageListBlock())], blank=True)
+
```

Comment:
Instead of a related model, I've used streamfields for this. The design shows too many variations

Reply:
![image](https://user-images.githubusercontent.com/2128707/80519193-8137ef80-897f-11ea-905b-5368e713f1fe.png)

Reply:
Good point, but this is a confident no - Pages and Landing pages wont have any actual relationship relationships